verksamhetsomraden/verksamhetsomraden.html

```diff
@@ -0,0 +1,368 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Verksamhetsområden - Myndigheten för yrkeshögskolan</title>
+    <script src="nav.js" defer></script>
+    <link rel="stylesheet" href="styles/flipCard.css">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="footer/main.css" />
+    <link rel="stylesheet" href="styles/main.css" />
+    <link rel="stylesheet" href="styles/verksamhetsomraden.css" />
+    <script src="https://kit.fontawesome.com/d7a96f613e.js" crossorigin="anonymous"></script>
+</head>
+    <body>
+
+      <header>
+        <nav class="nav">
+            <div class="logo"><a href="../startsida/index.html"><img src="./images/Logo.svg" alt="Logo"></a></div>
+            <ul id="normal-nav">
+                <li><a href="../startsida/index.html">Start</a></li>
+                <li><a href="../om-oss/index.html">Om oss</a></li>
+                <li><a href="./verksamhetsomraden.html">Verksamhet</a></li>
+                <li><a href="../regelverk/index.html">Regelverk</a></li>
+                <li><a href="../statistik/index.html">Statisitk</a></li>
+                <li><a href="../publikationer/index.html">Publikationer</a></li>
+                <li><a href="../nyhetsrum/index.html">Nyhetsrum</a></li>
+                <li><a href="../kontakt/index.html">Kontakt</a></li>
+                <li><a id="mina-sidor" href="#">Mina sidor</a></li>
+                <li class="anpassing"><img src="./images/ic_insert_comment_24px.svg" alt="chat"><img
+                        src="./images/ic_brightness_6_24px.svg" alt="brightness"><img
+                        src="./images/ic_format_size_24px.svg" alt="size"></li>
+            </ul>
+            <button class="fas fa-bars" id="hamburger" onclick="dropDown()"></button>
+        </nav>
+    </header>
+
+<!--main-->
+        <div class="s-wrapper">
+
+            <div class="box box1">
+                 <div class="s-startsida">
+                    <h1>Verksamhetsområden</h1>
+                    <br />
+                    <h3>Myndigheten för yrkeshögskolan har i 
+                    uppdrag att ansvara för frågor som rör 
+                    utbildningar inom yrkeshögskolan.
+                    Dessutom ansvarar vi för ytterligare 
+                    frågor inom utbildningsområdet.</h3>
+                </div>
+            </div>
+
+            <div class="box box20">
+              <form id = "searchFunctionMain" action="sok.html" method="GET">
+                <div class="searchWrapper">
+                    <input id = "searchBar" name="q" type="search" placeholder="Ange sökord" />
+                    <i id = searchFunctionIcon class="fas fa-search" style='font-size:40px'></i>
+                </div>
+            </form>
+            </div>
+
+
+            <div class="box box13">
+                <div class="yrkesimages">
+                  <img src="images/Yrkeshogskolan.PNG" alt="Yrkeshogskolan" />
```

Comment:
Can't load images, getting a 404 not found error. Happens in both Chrome and Firefox, the issue seems to be related to uppercase image extension. Changing it to lowercase seems to fix the issue for me.

The problem is found in these lines; 80, 94, 108, 123 and 137.